mod/Jailbreak.LastGuard/LastGuard.cs

```diff
@@ -65,26 +65,25 @@ public int CalculateHealth()
 
     public void StartLastGuard(CCSPlayerController lastGuard)
     {
-        var ctPlayerPawn = lastGuard.PlayerPawn.Value;
+        var guardPlayerPawn = lastGuard.PlayerPawn.Value;
 
-        if (ctPlayerPawn == null || !ctPlayerPawn.IsValid) return;
+        if (guardPlayerPawn == null || !guardPlayerPawn.IsValid) return;
 
-        var ctHealth = ctPlayerPawn.Health;
-        var ctCalcHealth = CalculateHealth();
+        var guardHealth = guardPlayerPawn.Health;
+        var guardCalcHealth = CalculateHealth();
 
-        ctPlayerPawn.Health = ctHealth > ctCalcHealth ? 125 : ctCalcHealth;
-        Utilities.SetStateChanged(ctPlayerPawn, "CBaseEntity", "m_iHealth");
+        guardPlayerPawn.Health = guardHealth > guardCalcHealth ? 125 : guardHealth;
```

Comment:
Looking at the logic again, I think it would be better (at least for balance) to just always set the CT's health the the calculated health that they should have.

Reply:
In CS:GO it wouldn't go lower than 100 iirc. I can take a look at the source though